styleguide/source/_patterns/01-molecules/06-components/13-subcategory-topics-description/_subcategory-topics-description.scss

```diff
@@ -0,0 +1,82 @@
+/**
+ * @file subcategory-topics-description.scss
+ *
+ * Copyright 2017 Palantir.net, Inc.
+ */
+
+// Subcategory item
+.subcategory-topics-description {
+  @include rule-horizontal(1px, $gray-8, solid);
+  margin-bottom: 40px;
+
+  @include breakpoint($bp-med) {
+    margin-bottom: 60px;
+  }
+}
+
+// Subcategory title and teaser wrap
+.subcategory-topics-description_wrap {
+  @include breakpoint($bp-med) {
+    @include grid__unit--cols(6);
+    padding-right: 15px;
```

Comment:
This padding seems a little small (on tablet and desktop widths)... the design doesn't specify, but this seems a little tighter than the implementation on production. Maybe bump it up to 40px, that's a common gutter, right?
<img width="1105" alt="pattern_lab_-_molecules-subcategory-topics-description" src="https://cloud.githubusercontent.com/assets/238201/25871023/b39da282-34cb-11e7-8063-566dae6e6c44.png">